### PR TITLE
Remove usages of deprecated old `native "<name>"` syntax.

### PR DESCRIPTION
**Pull request by mkustermann**
The old `native "<name>"` syntax is being replaced with external functions that
have a `@pragma("external-name", "<name>")`.

See https://github.com/dart-lang/sdk/issues/28791

### CODE REVIEW

File: shell/platform/fuchsia/dart-pkg/zircon/lib/src/handle.dart [hunk 1]
```diff
@@ -17,33 +17,40 @@ class Handle extends NativeFieldWrapperClass1 {
...
+  external int get koid;
 
   @override
   String toString() => 'Handle($handle)';
 
   @override
   bool operator ==(Object other) {
-    return other is Handle
-        && other.handle == handle;
+    return other is Handle &&
```
Comment (dcharkes) [inline shell/platform/fuchsia/dart-pkg/zircon/lib/src/handle.dart:34]: spurious formatting?

**Review by dcharkes — APPROVED** [review]
lgtm

**Review by mraleph — APPROVED** [review]

File: shell/platform/fuchsia/dart-pkg/zircon/lib/src/handle.dart (hunk 1 above)
Comment (mkustermann) [reply shell/platform/fuchsia/dart-pkg/zircon/lib/src/handle.dart]: Yeah, I changed the code, it autoformatted, then I had to undo the autoformatting by-hand, ...

(Flutter doesn't use `dart format` for reasons beyond my comprehension)

Comment (dcharkes) [reply shell/platform/fuchsia/dart-pkg/zircon/lib/src/handle.dart]: > (Flutter doesn't use `dart format` for reasons beyond my comprehension)

++

**Pull request merged**